### DISCUSSION TITLE
搞笑諾貝爾獎

### DISCUSSION THREAD

**Discussion by sorn32** [1101-GS4719-C/Python-Community-GS4719-C#1649] [General]
突然想到這個是因為看到一篇章魚燒研究的文章，作者利用數學公式，來推斷圓型章魚燒的半徑上限，甚至還寫成論文! 真的有種浪費才能的感覺，說不定會是下一屆搞笑諾貝爾獎得主✧◝(⁰▿⁰)◜✧
我對前幾屆的發表，有幾個印象特別深刻，像是用流體力學來探討貓能否同時處於固態和液態的問題、研究以坐姿自我進行大腸鏡檢查等等，大家有其他覺得有趣的嗎?

**Comment by Chengbosco** [comment]
我覺得這屆最有趣的是機械工程獎，他們利用蜘蛛腳收縮的結構，藉由改變蜘蛛體內壓力來使其腳張開或收縮，創造出以蜘蛛為體的機械爪來抓取不規則形狀的物體。這讓我想到之前看過正在研究的火星攀岩車(https://youtu.be/qaPXkDnVhUE?t=225)，也是利用收縮來攀岩，或許可以稍加運用也說不定。

**Comment by eric202501** [comment]
分享一個有趣的東西，搞笑諾貝爾獎頒獎典禮會邀請一些科學家談他們的認真研究，並且只給他們講兩次，第一次給 24 秒讓科學家講解他2的研究（會安排人認真計時，並且會瘋狂提醒他們），第二次他們只能講 7 個字總結自己的研究，其實蠻好笑的。

**Comment by mrweixd** [comment]
我印象最深刻的是2022發表的「大象表面積公式」： S=8.245+6.807H+7.073FFC
H指地到大象肩膀的高度，FFC是大象前腿的直徑。
以及有人成功證明讓牛or馬往樓梯上走比往下走容易很多。

**Comment by William08290829** [comment]
我一直都覺得這個獎項很有趣，可以讓人看到在嚴謹的科學之下也有有趣的一面，讓大眾知道說科學研究不是無聊的理論書面或是數學而已。而且他們雖說是搞笑，但其嚴謹的程度不會輸一般的諾貝爾獎，其搞笑的研究最後得到的結論其實有些也是能發人省思的。

**Comment by thomasxshen** [comment]
我印象最深刻的是，有人用大便做出了一把大便刀，更扯的是還能切東西，也不禁讓我反思人類的文明是不是在倒退，做出這種大便刀對於社會一點用處都沒有，還很快就融化了，即使在資源短缺的地方他依然毫無用處，超級可悲。

**Comment by YuanWang08** [comment]
我也同感覺這個獎項十分的有趣，可以讓社會大眾看到在嚴謹的科學之下既然也有有趣之處，讓社會大眾知道這科學研究定不只是無聊的道理或真理而已。況且，科學家們自己雖說是搞笑，但其實還是很嚴謹的，不會輸一般的諾貝爾獎，其搞笑的研究最後得到的結論其實有些也是能發人省思的。

**Comment by titan92601** [comment]
我認為搞笑諾貝爾獎的得獎研究總是充滿創意和幽默感。印象深刻的研究包括用流體力學探討貓是否同時處於固態和液態、自我進行大腸鏡檢查、研究磁鐵對鴿子導航的影響、以及披薩的健康效應等。這些研究看似搞笑，卻展現了科學家們的創意思維和探索精神，每年都能帶來不少歡笑與驚喜。

**Comment by SusuTima** [comment]
我最喜歡的是，
傳播獎：嗎話說著倒能你。
這個獎項頒給了一群科學家，他們調查了擅長倒著說話的人，包括分析神經成像。
我覺得蠻酷的，因為平常說話不會往這個方向去想，有點打破了我的眼界。

**Comment by alexender0104** [comment]
我印象最深刻的是去年的醫學獎，表彰其利用屍體來研究人的兩個鼻孔中的毛髮數量是否相等，我認為這是一件很無俚頭的行為，不知道是誰腦洞大開，提出這個奇葩的想法，並也付諸行動!

**Comment by hbh10601** [comment]
我印象最深的是2023年的文學獎：重複寫字，直到感覺不對勁
我有時候常常一直看同一個字，看到覺得自己不認識這個字了。原本以為只有我是這樣覺得，後來看到這個文學獎
他的解釋如下：

上述的情境，稱為「猶昧感」（Jamais Vu），「猶昧感」是「既視感」（Deja Vu）的反義詞，描述人們對熟悉的事物，突然感到陌生，也是這篇論文主要探討的主題。

這研究的笑點在於他的實驗，他們讓受試者一直重複寫同一個字，跟小學被老師罰抄生字一樣。

實驗中，約有三分之二的受試者體驗到「猶昧感」，這些受試者大約在重複 30 次或一分鐘後開始感到異狀。另外，研究也發現平常越容易發生「既視感」的人，也更容易發生「猶昧感」，未來「猶昧感」的相關研究，可能會加深我們對「既視感」的理解。

**Comment by Shang621** [comment]
搞笑諾貝爾獎將幽默和科學結合，表彰那些以輕松和幽默的方式推動科學進步的人。獎項可頒發給提出荒誕但具有創新性的研究，或是以滑稽方式傳達科學知識的科學傳播者。這樣的獎項不僅能增添科學界的趣味性，也能吸引更多人對科學感興趣。同时也提醒人們科學研究不一定要過於嚴肅，也可以搞笑和有趣
